apps/emails-and-messages/src/modules/mjml/configuration/mjml-config-input-schema.ts

```diff
@@ -0,0 +1,77 @@
+import { z } from "zod";
+import { messageEventTypes } from "../../event-handlers/message-event-types";
+import { smtpEncryptionTypes } from "./mjml-config";
+
+export const mjmlConfigInputSchema = z.object({
```

Comment:
not related to this file, but in general. Can you please save config in metadata with a version prefix? Eg. if app is `1.0.0` lets set it with a key `app_config_1.0.0` or something like that?

We need to do that, because when we change schema, we will have huge problems with migrations. With this versions, you could write code with following algorithm

1. Fetch newest schema (eg 1.2.0)
2. If not found, fetch older schema (1.0.0)
3. Make in-memory mapping to new schema
4. Save schema to 1.2.0
5. Fetch new schema
6. Cleanup

Reply:
Also, instead of exporting all schemas loose, we can do something like 

`export const AppCofnfig_1_0_0 = {mjmlConfigurationEventObject, ...}`

So in app we are not mixing them

Reply:
It could be implemented, but I'm a bit afraid of the lack of transactions.
Also, the configuration types should be fully implemented as Zod schemas, so we'll be able to parse it from incoming webhooks. Will also be helpful during migrations to make sure data are in the shape we expect them to be.